packages/examples/src/wrapperAdvanced.ts

```diff
@@ -37,7 +37,7 @@ Same again.`
             $type: 'WebSocket',
             name: 'wrapper42 language client',
             host: 'localhost',
-            port: 3000,
+            port: 30000,
```

Comment:
30000 instead of 3000? Conflicts with other stuff?

Reply:
Yes, 3000 clashes, because we use it in multiple projects.